src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+}
```

Comment:
```suggestion
export interface User {
  id: number;
  name: string;
  username: string;
  email: string;
}
```

use semicolon instead of comma